{{cookiecutter.project_dirname}}/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -310,9 +310,13 @@ class Testing(ProjectDefault):
     # unittest-xml-reporting (aka xmlrunner)
     # https://github.com/xmlrunner/unittest-xml-reporting#django-support
 
-    TEST_OUTPUT_FILE_NAME = "report.xml"
+    PYTHONBREAKPOINT = values.Value(environ_name="PYTHONBREAKPOINT", environ_prefix="")
 
-    TEST_RUNNER = "xmlrunner.extra.djangotestrunner.XMLTestRunner"
+    if PYTHONBREAKPOINT is None:  # pragma: no cover
```

Comment:
Instead of introducing `if PYTHONBREAKPOINT is None`  condition in settings.py, we can consider to override the `TEST_RUNNER` variable from orchestrator 

_settings.py_
`TEST_RUNNER = values.Value("xmlrunner.extra.djangotestrunner.XMLTestRunner")`

on the orchestrator:

_.env_template_
`DJANGO_TEST_RUNNER=django.test.runner.DiscoverRunner`

_docker_compose.yaml_
```
backend:
  ...
  environment:
    ...
    - DJANGO_TEST_RUNNER
    ...
```

Reply:
@trottomv I would love to see a full-fledged proposal from yours in a separate PR. This way, we are able to visualize your proposal in a more organic way and discuss it.